packages/components/radios/README.md

```diff
@@ -75,36 +75,28 @@ To discuss or contribute to this component, visit the [GitHub issue for this com
 
 ```html
 <div class="nhsuk-form-group">
-
-  <fieldset class="nhsuk-fieldset" aria-describedby="example-hint">
-
-  <legend class="nhsuk-fieldset__legend">
-    Have you changed your name?
-  </legend>
-
-  <span class="nhsuk-hint" id="example-hint">
+  <fieldset class="nhsuk-fieldset" aria-describedby="example&#39;-hint">
```

Comment:
Dodgy character - think this was fixed here?
https://github.com/nhsuk/nhsuk-frontend/pull/354